Cargo.lock.bak

```diff
@@ -1,447 +0,0 @@
-[[package]]
```

Comment:
don't forget to clean this up before merging

Reply:
I understand that @lucksus added this file in develop 3 days ago, probably by mistake. I'm erasing it here. So I'm not sure what you are asking. Should I leave it?

Reply:
oh! yes i see, good job then 👍

Reply:
Oh! Ooops.. sorry. Yes, this needs deleted..